workitem/event/event_repository_blackbox_test.go

```diff
@@ -95,6 +96,25 @@ func (s *eventRepoBlackBoxTest) TestList() {
 		assert.Equal(t, fxt.Identities[0].ID.String(), eventList[0].New.([]interface{})[0])
 	})
 
+	s.T().Run("event description", func(t *testing.T) {
+		oldDescription := rendering.NewMarkupContentFromLegacy("description1")
+		fxt := tf.NewTestFixture(t, s.DB, tf.WorkItems(1, func(fxt *tf.TestFixture, idx int) error {
+			fxt.WorkItems[idx].Fields[workitem.SystemDescription] = oldDescription
+			return nil
+		}))
+		newDescription := rendering.NewMarkupContentFromLegacy("description2")
+		fxt.WorkItems[0].Fields[workitem.SystemDescription] = newDescription
+		wiNew, err := s.wiRepo.Save(s.Ctx, fxt.WorkItems[0].SpaceID, *fxt.WorkItems[0], fxt.Identities[0].ID)
+		require.NoError(t, err)
+		eventList, err := s.wiEventRepo.List(s.Ctx, fxt.WorkItems[0].ID)
+		require.NoError(t, err)
+		require.NotEmpty(t, eventList)
+		require.Len(t, eventList, 1)
+		require.Empty(t, eventList[0].Old)
+		require.Empty(t, eventList[0].New)
```

Comment:
These empty assertions will not work. We can check for equality:
```
               require.Equal(t, "description1", eventList[0].Old)
               require.Equal(t, "description2", eventList[0].New)
```

Reply:
Thanks Baiju.I've updated the PR. Please see https://github.com/baijum/fabric8-wit/pull/3/commits/44d8832abcf3894e40cab060e50f78010f508223